test/widgets/content_test.dart

```diff
@@ -93,6 +93,18 @@ void main() {
     tester.widget(find.text(r'\lambda'));
   });
 
+  testWidgets('GlobalTime smoke', (tester) async {
```

Comment:
nit: put this after the MathInlineNode test below, to match the code under test in `lib/widgets/content.dart`